Autopilot/Src/SensorFusion.cpp

```diff
@@ -86,3 +86,16 @@ SFError_t SF_GetAttitude(SFAttitudeOutput_t *Output, IMU_Data_t *imudata, Airspe
     return SFError;
 }
 
+double getTrack(double latitudeSpeed, double longitudeSpeed)
```

Comment:
Like I say, not like this. User should not have to call this. Track is just another datapoint bundled in the sensor fusion struct.